src/common/lockdep.h

```diff
@@ -30,4 +33,17 @@ extern int lockdep_will_lock(const char *n, int id, bool force_backtrace=false,
 extern int lockdep_locked(const char *n, int id, bool force_backtrace=false);
 extern int lockdep_will_unlock(const char *n, int id);
 extern int lockdep_dump_locks();
+
+#else
+
+static constexpr bool g_lockdep = false;
+static inline int lockdep_register(const char*) { return 0; }
+static inline void lockdep_unregister(int id) {}
+static inline int lockdep_will_lock(const char*, int id, bool, bool) { return  id; }
```

Comment:
This parameters should be the same with:

```
extern int lockdep_will_lock(const char *n, int id, bool force_backtrace=false,
			     bool recursive=false);
```

Or we are hitting the following errors:

```
../src/common/RWLock.h:106:70: error: too few arguments to function 'int lockdep_will_lock(const char*, int, bool, bool)'
  106 |     if (lockdep && g_lockdep) id = lockdep_will_lock(name.c_str(), id);
      |                                                                      ^
```

Reply:
@lxbsz fixed and repushed.